metadata.py

```diff
@@ -0,0 +1,68 @@
+import json
```

Comment:
We will rewrite the whole module later as a standalone client for communication with www.ebi.ac.uk